packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -22,6 +22,8 @@ const restrictedNodeFields = [
   `id`,
   `internal`,
   `parent`,
+  `Entity`,
+  `Reference`,
```

Comment:
This restricts field names, but we need to restrict Content Type names

Reply:
I was analyzing the code, but I could not figure out how to restrict the Content Type names. Could you give a hint?

Reply:
I don't know either :D